frontend/src/main/scala/bloop/Project.scala

```diff
@@ -36,8 +39,15 @@ case class Project(name: String,
     properties.setProperty("allScalaJars",
                            scalaInstance.allJars.map(_.getAbsolutePath).mkString(","))
     properties.setProperty("tmp", tmp.syntax)
+    properties.setProperty("fork", javaEnv.fork.toString)
+    properties.setProperty("javaHome", javaEnv.javaHome.syntax)
+    properties.setProperty("javaOptions", javaEnv.javaOptions.mkString(";"))
     properties
   }
+
+  lazy val fullClasspath: Array[AbsolutePath] = {
```

Comment:
Why is this lazy?

Reply:
I would do it the other way around -- I would call this classpath, and then I would call the other version without the classes dir `rawClasspath`. I'm saying this because most of the times we want the `fullClasspath`, not the classpath, and I can see that biting us in the future.

Reply:
Sounds good